### ISSUE TITLE
Facility Locator QA/508 Check

### ISSUE THREAD

**Issue by mmiddaugh** [department-of-veterans-affairs/va.gov-team#9555]
# Team Name
Facilities

# Product/Feature Name
Facility Locator

## Checkpoint Stage
- [x] Other: Requesting 508 and QA review for a variety of Facility Locator 508 defect resolutions and small UI enhancements which are now available in **Production**

## Point of contact
Michelle Middaugh

## Production URL 
https://www.va.gov/find-locations
No credentials needed 

## User stories for testing

[#5620](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5620)
Given a Veteran visits https://www.va.gov/find-locations: and initiates a search of any facility type which yields results:
- [ ] Sighted veterans can view alpha pins correlating facility locations on the map to those within the search results list. 
- [ ] Veterans with assistive devices are presented with information linking the card ID with droplet/pin on map (aria-describedby)

[#5555](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5555)
Given a Veteran visits https://www.va.gov/find-locations using a screen reader device and initiates a search of any facility type yielding results:
- [ ] Search results lists are presented as ordered list with contextual information for Veterans using screen reader devices. 

#[5501](https://github.com/department-of-veterans-affairs/va.gov-team/issues/5501)
- [ ] Sighted veterans can view a pin marking their geographic location as specified in the search parameters so they know their location relevant to search results. 

[#7244](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/7244)
Given a Veteran has visited https://www.va.gov/find-locations with a screen reader device, entered a geographic location and performed a search with facility type = Urgent care and service type = Community urgent care, 
- [ ] then the "Important information about your Community Care appointment" alert is read as  "heading level 2, Important information about…" so that the page structure is understood. 

[#7288](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/7288)
- [ ] When a screenreader device reads search results on https://va.gov/find-locations, the user should be able to navigate away from the search results using a keyboard. 

[#7291](https://app.zenhub.com/workspaces/vft-59c95ae5fda7577a9b3184f8/issues/department-of-veterans-affairs/va.gov-team/7291)
Given a Veteran uses the Firefox browser in Windows 10 to visit https://www.va.gov/find-locations with a screen reader device:
- [ ] The Veteran should be able to navigate to "Choose a VA facility type" dropdown using keyboard only 
- [ ] The Veteran should be able to traverse the options in the dropdown using keyboard only

## Next steps

- [ ] If the linked issue within the test case above is resolved, the QA or Accessibility specialist will close the related issue.
- [ ]  If the linked issue within the test case above remains unresolved, the QA or Accessibility specialist will comment in the related issue.
- [ ]  When all user stories above have been retested, the QA or Accessibility specialist will comment in this issue to confirm completion, tagging @mmiddaugh so that the team can coordinate any needed followup.
- [ ]  If additional work is needed, the Facilities team will coordinate another review after work is updated
- [ ]  Once all issues are addressed, the ticket is closed with a final comment, “All issues are addressed”, @-comment developer who opened issue, and close ticket/issue.
